src/journal-remote/journal-upload.c

```diff
@@ -208,6 +209,13 @@ int start_upload(Uploader *u,
                         return log_error_errno(SYNTHETIC_ERRNO(ENOSR),
                                                "Call to curl_easy_init failed.");
 
+                /* If configured, set a timeout for the curl operation. */
+                if (arg_network_timeout_usec != USEC_INFINITY) {
+                        easy_setopt(curl, CURLOPT_TIMEOUT,
+                                    (long) DIV_ROUND_UP(arg_network_timeout_usec, USEC_PER_SEC),
+                                    LOG_ERR, return -EXFULL);
+                }
```

Comment:
please drop the {} around the if block here, as per our coding style we avoid them if there's only a single expression used inside the {}, like it is here